doc/how-to-source-packages.rst

```diff
@@ -0,0 +1,309 @@
+How to get package *source code*
```

Comment:
```suggestion
How to use package *source code*
```

Reply:
I went with "How to locate and get package *source code* and reworked the introductory two sentences of that section.

Reply:
Thought that "locate and get" was too long. Considered "muster" but the meaning I'm familiar with is NZ/AU only so went with "shepherd".